cypress/integration/main_spec.js

```diff
@@ -0,0 +1,81 @@
+import { O_TRUNC } from "constants";
+import { getFriendlyExpiryDescriptionForHistory } from "../../src/lib/utils";
+
+describe('Probe Dictionary Main Component', function() {
```

Comment:
Not a big deal, obviously. But arrow functions are an option here.

```javascript
describe('Probe Dictionary Main Component', () => {
```

Reply:
Yep not consistent with that...will fix throughout.